README.md

```diff
@@ -128,6 +128,7 @@ NOTE: PWD hooks are currently not supported for POSIX shells.
 - `$_ZO_EXCLUDE_DIRS`: list of directories separated by platform-specific
   characters ("`:`" on Linux and macOS, and "`;`" on Windows) to be excluded from
   the database
+- `$_ZO_FZF_ARGS`: extra arguments to pass to fzf, e.g. `--height 25`
```

Comment:
```suggestion
- `$_ZO_FZF_ARGS`: extra arguments to pass to `fzf`
```

We don't have examples for any of the other env vars, so I don't think we need one here. Also, backtick-quote `fzf` because it's a command ;)